ozaria/site/components/interactive/PageInteractive/index.vue

```diff
@@ -80,23 +124,20 @@
         this.loading = true
 
         try {
-          this.interactive = await getInteractive(this.interactiveIdOrSlug)
-          this.interactiveType = this.interactive.interactiveType
-          if (!this.interactiveType) {
-            return Promise.reject('Interactive type is not set for the interactive ' + this.interactiveIdOrSlug)
-          }
-          if (!me.isSessionless()) { // not saving progress/session for teachers
-            const getSessionOptions = {
+          const interactivePromise = this.loadInteractive(this.interactiveIdOrSlug)
+          const interactiveSessionPromise = this.loadInteractiveSession({
```

Comment:
Need to load session only if `!me.isSessionless()` - https://github.com/codecombat/codecombat/pull/5384/files#diff-ae85ddaad26013f9e387dc50ef3bc7a5L88